src/main/external-resources/UMS.conf

```diff
@@ -196,7 +196,7 @@ http_engine_v2 =
 # Prevent system idle sleep
 # -------------------------
 # If your UMS host has a power-saving mode, use this option to prevent the
-# system from hibernating or sleeping. Currently only Windows and macOS are
+# system from hibernating or sleeping. Currently only Windows and Mac OS are
 # supported. There are three modes:
```

Comment:
It's too much for a dinosaur like me LOL
I will stick with Mac OS, but now i leaned that they changed name, as the political parties in my country when things go wrong.